### PR TITLE
chore(docs): add mdx-embed guide

### PR DESCRIPTION
**Pull request by PaulieScanlon**
Add MDX Embed installation and links to readme

### CODE REVIEW

**Review by ChristianMurphy — CHANGES_REQUESTED** [review]
Thanks @PaulieScanlon!
Could we move this to https://github.com/mdx-js/awesome? Or maybe to a guide? (maybe both)

the main readme points folks towards where to learn basics, intentionally concise, linking out to guides, rather than being comprehensive.

mdx-embed does meet the 30+ day and 50+ star requirements for https://github.com/mdx-js/awesome and would I be happy to see it as an awesome community project :sparkles:

**Comment by johno** [comment]
Following along with @ChristianMurphy, I'd love to see this in docs/guides/mdx-embed as its own file (Which we will add nav to before v2 goes live), aside from that it looks great!

**Comment by PaulieScanlon** [comment]
Hi @johno and @ChristianMurphy I've moved this to docs/guides/mdx-embed and added a link to the new page in docs/guides/index

Happy to make further changes if you like.

**Review by johno — APPROVED** [review]
Thank you @PaulieScanlon!

File: docs/guides/mdx-embed.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```javascript
+import React from 'react'
+import {MDXEmbedProvider} from 'mdx-embed'
+
+export default props => <MDXEmbedProvider>{props.children}</MDXEmbedProvider>
+```
+
+### Usage
+
+```javascript
```
Comment (ChristianMurphy) [inline docs/guides/mdx-embed.md:24]:
```suggestion
```mdx
```

**Review by ChristianMurphy — APPROVED** [review]

**Pull request merged**